Sources/PostgresNIO/New/Data/JSON+PSQLCodable.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 14.01.21.
+//
+
+import NIOFoundationCompat
```

Comment:
Ditch Xcode template headers.
```suggestion
import NIOFoundationCompat
```